docs/farming-&-staking/farming/advanced-cli/cli-tips.mdx

```diff
@@ -53,6 +53,31 @@ In general you should be able to download the latest release, and re-start the N
 
 There are some cases where version updates will cause issue with your Node & Farmer and you may have to wipe & purge your node, typically when errors occur. If you have any issues you can always check our [Forums](https://forums.subspace.network) and hop in our [Discord](https://discord.gg/subspace-network) Server to ask for help.
 
+### Wipe & Purge
+
+If you were running a node previously, and want to switch to a new snapshot, please perform these steps and then follow the guideline again:
+```bash
+# Replace `FARMER_FILE_NAME` with the name of the farmer file you downloaded from releases
+./FARMER_FILE_NAME wipe PATH_TO_FARM
+# Replace `NODE_FILE_NAME` with the name of the node file you downloaded from releases
+./NODE_FILE_NAME purge-chain --chain gemini-3g
+```
+Does not matter if the node/farmer executable is the previous one or from the new snapshot, both will work :)
+The reason we require this is, with every snapshot change, the network might get partitioned, and you may be on a different genesis than the current one.
```

Comment:
I'm not sure the snapshot language is as relevant as it used to be. I believe that now we are at a level of stability where a wipe/purge will only be required in extreme situations. Certainly not as often as earlier testnet stages.

Keen to hear if @nazar-pc has any thoughts on talking about snapshots like this still.

Reply:
I agree, during whole 3g there were there were just a few one-offs where specific farmers had to wipe things. For everyone else there should be no issues that require starting from scratch and only transitioning to the next major version of the network requires this.

Reply:
Let's reserve wipe/purge for switching networks or recovering from catastrophic data corruption. The former should be non-existent after mainnet but possibly useful for parallel testnets. The latter will hopefully be rare but there could be users with stability issues it will help.